antora-ui-camel/src/css/frontpage.css

```diff
@@ -74,30 +74,32 @@ header.frontpage p {
   line-height: 1.7rem;
 }
 
-.frontpage a.button-dark,
-.frontpage a.button-dark:hover {
+a.button,
+section.frontpage a.button {
   white-space: nowrap;
-  padding: 0.7rem 1rem;
-  background-color: var(--color-camel-orange);
-  background-image: none;
-  background-repeat: no-repeat;
-  background-position: 0;
-  color: white;
-  font-weight: bold;
   margin-right: 0.5rem;
+  background-image: none;
   border-radius: 3rem;
-  outline: none;
-  display: inline-block;
-  box-shadow: 0 4px #8e480b;
+  font-weight: bold;
+  padding: 0.4rem 1rem;
```

Comment:
```suggestion
  padding: 0.4rem 1rem;
  line-height: 2.5rem;
```

Reply:
When exactly are you observing the overlap? I have already provided line-height for <1024px screens to avoid the overlap, and I can't see the buttons overlapping for any screen width.

Reply:
This overlapping is observed for 1024px as well.

Reply:
My view at 1024px 100% zoom

![Screenshot 2020-08-28 at 1 48 47 PM](https://user-images.githubusercontent.com/32356795/91538379-64172b80-e935-11ea-8f54-27f8350d24c6.png)